ast/policy.go

```diff
@@ -154,10 +155,16 @@ type (
 
 	// Annotations represents metadata attached to other AST nodes such as rules.
 	Annotations struct {
-		Location *Location           `json:"-"`
-		Scope    string              `json:"scope"`
-		Schemas  []*SchemaAnnotation `json:"schemas,omitempty"`
-		node     Node
+		Location         *Location                    `json:"-"`
+		Scope            string                       `json:"scope"`
+		Title            string                       `json:"title,omitempty"`
+		Description      string                       `json:"description,omitempty"`
+		Organizations    []string                     `json:"organizations,omitempty"`
+		RelatedResources []*RelatedResourceAnnotation `json:"related_resources,omitempty"`
+		Authors          []*AuthorAnnotation          `json:"authors,omitempty"`
+		Schemas          []*SchemaAnnotation          `json:"schemas,omitempty"`
+		Custom           map[string]interface{}       `json:"custom,omitempty"`
```

Comment:
Do we get into trouble with ordering here? Golang's map iteration is randomized, and we'll have to print these somewhere, like in `opa fmt` (= the `format` package)

Reply:
Ah! Didn't think about that. Will be very annoying if your annotations kept being reordered by `fmt` ..
I suppose we could accept a YAML map in the parser, but pull that into a list of `CustomAnnotation{Key,Value}` here, sorted by key. But that wouldn't handle nested maps :(. We could recursively transform nested maps to a sorted key/value list .. but that might be annoying to SDK users, unless we turn it back to a map for them ..
Thoughts?

Reply:
Another option is to sort the output for `fmt`. Will need to look into how that works, to see if that is viable.

Reply:
Ignore me, for the formatter, it's only meaningless comments...

Reply:
Good :). I was hoping that would be the case. Was the first thing on my agenda today to check.

Reply:
Confirmed, `fmt` just reprints comments as-is.